YvesIraguha/TeaHouse
# #169673380 Admin Login

#### What does this PR do?
- Implement admin login
#### Description of Task to be completed?
- Validate email and password on login
-  Return token on successful login
-  Create seeds for creating admin
#### How should this be manually tested?
-  Clone the repo and checkout the branch `admin-login-169673380`
-  Run `npm i` to install all dependencies
-  Ensure that you have all environment variables specified in this file `.env.sample`
-  Run `npm run test` to run tests, they should pass. 
-  Run `npm run start` to start the server.
-  Use client tool, preferably postman, and consume `/api/v1/auth/login` endpoint with correct `email` and `password` you should see the outcome. 
#### Any background context you want to provide?
Admin is created through seeds 
#### What are the relevant pivotal tracker stories?
 #[169673380](https://www.pivotaltracker.com/story/show/169673380)
